examples/devices/remote_controlled.py

```diff
@@ -85,138 +84,138 @@ def __init__(
+        """A regex bytes command which returns the byte encoded value of unobserved.
 
         Returns:
-            bytes: The big endian float encoded value of unobserved
+            bytes: The big endian float encoded value of unobserved.
         """
         return struct.pack(">f", self._device.unobserved)
 
     @RegexCommand(r"U", format="utf-8")
     async def get_unobserved_str(self) -> bytes:
-        """A regex string command which returns the utf-8 encoded value of unobserved
+        """A regex string command which returns the utf-8 encoded value of unobserved.
 
         Returns:
-            bytes: The utf-8 encoded value of unobserved
+            bytes: The utf-8 encoded value of unobserved.
         """
         return str(self._device.unobserved).encode("utf-8")
 
     @RegexCommand(b"\x02(.{4})")
     async def set_unobserved_bytes(self, value: bytes) -> bytes:
-        """A regex bytes command which sets and echos the value of unobserved
+        """A regex bytes command which sets and echos the value of unobserved.
 
         Args:
-            value (bytes): The new big endian float encoded value of unobserved
+            value (bytes): The new big endian float encoded value of unobserved.
 
         Returns:
-            bytes: The big endian float encoded value of unobserved
+            bytes: The big endian float encoded value of unobserved.
         """
         self._device.unobserved = struct.unpack(">f", value)[0]
         return struct.pack(">f", self._device.unobserved)
 
     @RegexCommand(r"U=(\d+\.?\d*)", format="utf-8")
     async def set_unobserved_str(self, value: float) -> bytes:
-        """A regex string command which sets and echos the value of unobserved
+        """A regex string command which sets and echos the value of unobserved.
 
         Args:
-            value (int): The new value of unobserved
+            value (int): The new value of unobserved.
 
         Returns:
-            bytes: The utf-8 encoded value of unobserved
+            bytes: The utf-8 encoded value of unobserved.
         """
         self._device.unobserved = value
         return "Unobserved set to {}".format(self._device.unobserved).encode("utf-8")
 
     @RegexCommand(chr(0x1F95A), format="utf-8")
     async def misc(self) -> bytes:
-        """A regex string command which returns a utf-8 encoded character
+        """A regex string command which returns a utf-8 encoded character.
 
         Returns:
-            bytes: A utf-8 encoded character
+            bytes: A utf-8 encoded character.
         """
         return chr(0x1F430).encode("utf-8")
 
     @RegexCommand(r"H=(\d+\.?\d*)", format="utf-8")
     async def set_hidden(self, value: float) -> None:
-        """A regex string command which sets the value of hidden
+        """A regex string command which sets the value of hidden.
 
         Args:
-            value (float): The new value of hidden
+            value (float): The new value of hidden.
```

Comment:
Is it possible to omit the type from the docstring if it is in the function signature or will pds complain? (Tom is curious)

Reply:
`pydocstyle` will allow it

Reply:
@thomascobb